shimmingtoolbox/cli/mask.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python3
+
+import click
+import nibabel as nib
+import numpy as np
+import os
+
+import shimmingtoolbox.masking.threshold
+from shimmingtoolbox.masking.shapes import shape_square
+from shimmingtoolbox.masking.shapes import shape_cube
+
+CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+
+
+@click.group(context_settings=CONTEXT_SETTINGS, help=f"Create a mask based on a specified shape (cube, square, etc.) "
+                                                     f"or based on the thresholding of an input image.")
+def mask_cli():
+    pass
+
+
+@mask_cli.command(context_settings=CONTEXT_SETTINGS,
+              help=f"Create a cube mask from the input file. Return the filename for the output mask.")
+@click.option('-input', 'fname_input', type=click.Path(), required=True, help="Input path of the nifti file to mask")
+@click.option('-output', type=click.Path(), default=os.curdir, help="Output folder for mask in nifti file")
+@click.option("-size", required=True, type=int, help="Length of the side of the cube along first, second and third "
+                                                     "dimension (in pixels)")
```

Comment:
every edge of a cube have the same length, so no need to mention the 1st 2nd and 3rd dim
```suggestion
@click.option("-size", required=True, type=int, help="Edge size of the cube (in pixels)")
```

Reply:
The mask is based off the API named cube which takes all 3 dimensions as inputs, the API could be named prism or rectangle to have a more appropriate name but to be consistent with the API I would leave it as is and revisit in another PR.

Reply:
i see. in that case i would not call it “cube” because this is wrong. How about “box”?

and “rect” for the 2d version

Reply:
Ok I will change the name of cube and square in this case!